# Publish all joint states -----------------------------------
joint_state_controller:
  type: joint_state_controller/JointStateController
  publish_rate: 50

# Position Controllers ---------------------------------------
joint1_position_controller:
  type: effort_controllers/JointPositionController
  joint: joint1
  pid: {p: 2000.0, i: 100, d: 500.0}
joint2_position_controller:
  type: effort_controllers/JointPositionController
  joint: joint2
  pid: {p: 50000.0, i: 100, d: 2000.0}
joint3_position_controller:
  type: effort_controllers/JointPositionController
  joint: joint3
  pid: {p: 20000.0, i: 50, d: 1000.0}
joint4_position_controller:
  type: effort_controllers/JointPositionController
  joint: joint4
  pid: {p: 80000.0, i: 200, d: 50.0}
joint5_position_controller:
  type: effort_controllers/JointPositionController
  joint: joint5
  pid: {p: 80000.0, i: 100, d: 50.0}
joint6_position_controller:
  type: effort_controllers/JointPositionController
  joint: joint6
  pid: {p: 80000.0, i: 200, d: 50.0}
joint7_position_controller:
  type: effort_controllers/JointPositionController
  joint: joint7
  pid: {p: 80000.0, i: 100, d: 50.0}
